### ISSUE TITLE
Can‘t switch ON No-Go lines

### ISSUE THREAD

**Issue by WibiNorg** [naofireblade/homebridge-neato#50]
Hey there!

Started playing with this plugin the last 2 days and I love it!
I stumbled upon a issue: 

- I can't turn ON the no-go lines, which I would love to be always on (which it's supposed to be, right?)

Cheers,

Wibi

**Comment by Jodameister** [comment]
Same here. Can't turn on "No Go Lines". And this Option is always OFF. Any Solution?

**Comment by naofireblade** [comment]
Enabling settings on the neato server is a bit strange. To save the nogo lines setting permanently, you need to start cleaning directly after enabling the nogo lines. If you do not start a cleaning, the setting will not be saved and resetted as soon as you reopen the homekit app.

Does that help?

**Comment by WibiNorg** [comment]
Yes, also figured that no-go lines do work if you clean right after. I made it work for me by making an automation in the shortcuts app: 1. Turn No-Go Lines on 2. Wait 1 second (to be sure) 3. Start cleaning. Works like a charm when you use Siri!

**Comment by naofireblade** [comment]
Maybe it would be more useful to store the setting in homekit and do not sync it from the neato servers?

**Comment by Midifreakz** [comment]
It's so odd that this won't work anymore. No-Go lines were always enabled when using this plugin. I only have a 'clean' button in Homekit. Since a month or 2 no go lines went disabled by default. Something happened on Neato's side?

**Comment by naofireblade** [comment]
The plugin will save options set in HomeKit in the next version.
